namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-dev/resources/rds.tf

```diff
@@ -0,0 +1,37 @@
+variable "cluster_name" {}
+variable "cluster_state_bucket" {}
+
+module "rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.5"
+
+  cluster_name               = "${var.cluster_name}"
+  cluster_state_bucket       = "${var.cluster_state_bucket}"
+  team_name                  = "${var.team_name}"
+  business-unit              = "${var.business-unit}"
+  application                = "${var.application}"
+  is-production              = "${var.is-production}"
+  environment-name           = "${var.environment-name}"
+  infrastructure-support     = "${var.email}"
+  db_engine                  = "postgres"
+  db_engine_version          = "10.6"
+  db_instance_class          = "db.t2.small"
+  db_allocated_storage       = "5"
+  db_name                    = "datacaptureservice"
+  db_backup_retention_period = "${var.db_backup_retention_period}"
+}

```

Comment:
Can you please add the `providers` block, see this for an example: 
https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/blob/master/example/rds.tf